assets/css/global.less

```diff
@@ -860,244 +858,613 @@ a.caret-right{
-.post-section-white {
-  font-family: sans-serif;
-  background-color: #fff;
-  position: relative;
-  color: #000;
-  padding: 2% 5%;
-  & p {
-    font-size: 16px;
-    letter-spacing: 0.42px;
-    line-height: 22px;
-    & code {
-      background-color: #e6f6fe;
-      color: #000;
-    }
-  }
-  & .post-title {
-    font-size: 27px;
-    padding: 0 5%;
-    & p {
-      font-size: small;
-      color: #736c6c;
-      font-weight: 600;
-      margin-bottom: 0px;
-    }
-  }
-  & .dlist {
-    padding: 0 4%;
+@media (min-width: 768px) {
+  .col-sm-2-5{
+      float: left;
   }
-  & a:link{
-    color: #07a7fc;
-    text-decoration: none;
+  .col-sm-2-5 {
+      width: 20.66666667%;
   }
-  & h2 {
-    border-bottom: solid 1px #07a7fc;
+}
+@guides-font-family:    'Open Sans', sans-serif;
+@guides-font:           #1e252f;
+@guides-primary-color: #06a6fd; 
+
+.guides{
+  font-family: @guides-font-family;
+  color: @guides-font;
+}
+
+.guides-input-placeholder {
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 600;
+  letter-spacing: -0.23px;
+  color: #0000;
+}
+
+.guides .section-hero {
+  &:extend(.section-bg-blue);
+  background: url('../img/bg-header-squares.png') center bottom no-repeat, linear-gradient(@brand-primary, #06A2FF);
+  .container {
+    margin-top: 7%;
+    margin-bottom: 20px;
   }
-  & h4, h3, h2,label, a {
-    color: #4e4c4c;
+  h1 {
+    font-size: 40px;
+    letter-spacing: 1.13px;
+    margin-bottom: 11px;
   }
-  & .aside-wrapper {
-    background: white;
-    padding-top: 30%;
-    line-height: 2.5em;
-    font-weight: bold;
-    position: absolute;
-    top: 0;
-    bottom: 0;
-    left: 0;
+  p {
+    font-size: 20px;
+    font-weight: inherit;
+    line-height: 1.5;
+    letter-spacing: 0.56px;
   }
-  & section {
-    margin: 3%;
+  @media (max-width: @screen-sm-max) {
+    margin-top: -11%;
   }
 }
 
-#all-guides {
-  & a {
-    color: #06a6fd;
-    font-weight: bold;
+.searchrow {
+  margin-top: 15px;
+  .input-group {
+    width: 100%;
+    height: 46px;
+  }
+  button {
+    border-bottom-left-radius: 8px;
+    border-top-left-radius: 8px;
+    min-height: 56px;
+    height: 56px;
+    &:hover, &:focus {
+      .no-transform;
+    }
+  }
+  input {
+    background: #f5f9fa;
+    border-top-right-radius: 8px;
+    border-bottom-right-radius: 8px;
+    min-height: 56px;
+    height: 56px;
+    border: none;
+    color: @guides-font; 
+    .no-transform;
+    &:active, &:focus {
+      .no-transform;
+    }
   }
-  & a:hover {
+}
+
+.no-padding {
+  padding: 0 !important;
+  margin: 0 !important;
+}
+
+.no-transform {
+  transform: none;
+  box-shadow: none;
+  outline: none !important;
+}
+
+.guides-input-box::-webkit-input-placeholder { .guides-input-placeholder; }
+.guides-input-box::-moz-placeholder          { .guides-input-placeholder; }
+.guides-input-box::-ms-placeholder           { .guides-input-placeholder;}
+.guides-input-box::placeholder               { .guides-input-placeholder;} 
```

Comment:
What's this stuff?

Reply:
Styling for the placeholders within the input box to cater for various browsers https://github.com/gruntwork-io/gruntwork-io.github.io/pull/147/files/6da8b60d99762bbf2bc0fd6c78719c20420c8dbf#diff-1e69babe6c50a615d3a6ef7b48cf7484R888